test/src/test/java/hudson/model/SlaveTest.java

```diff
@@ -214,6 +215,7 @@ public void retentionFiltering() throws Exception {
     @Test
     @Issue("JENKINS-36280")
     public void propertyFiltering() throws Exception {
+        j.jenkins.setAuthorizationStrategy(new ProjectMatrixAuthorizationStrategy()); // otherwise node descriptor is not available
```

Comment:
Do we really need a `matrix-auth` dep in our tests? I would rather get rid of altogether if we can. A subject for another PR, presumably.